ols-1.md

```diff
@@ -52,15 +53,15 @@ Open Science skills
 **Outcome**: Ambassadors for Open Science practice, training and education across multiple European
 and international bioinformatics communities.
 
-**Process**: A 15-week mentoring & training program, based on the [Mozilla Open Leader program](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/), helping participants in becoming Open Science ambassadors by using three principles:
+**Process**: A 15 week mentoring & training program, based on the [Mozilla Open Leader program](https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/), helping participants in becoming Open Science ambassadors by using three principles:
```

Comment:
please keep the hyphen here and in other places

Reply:
will do!